configs/api_local_config.properties

```diff
@@ -0,0 +1,17 @@
+RUNNER=distribute
+FRAMEWORK=cucumber
+RUNNER_LEVEL=methods
+APP_NAME=apiPlatformTests
+ENVIRONMENT_CONFIG_FILE=./src/test/resources/environments.json
+IS_VISUAL=false
+LOG_DIR=target
+LOG_PROPERTIES_FILE=./src/test/resources/log4j.properties
+PARALLEL=1
+PLATFORM=api
+PROXY_KEY=HTTP_PROXY
+REPORT_PORTAL_FILE=src/test/resources/reportportal.properties
+RUN_IN_CI=false
+TARGET_ENVIRONMENT=prod
+LAUNCH_NAME_SUFFIX= on 'prod' Environment
+TEST_DATA_FILE=./src/test/resources/testData.json
+CLEANUP_DEVICE_BEFORE_STARTING_EXECUTION=false
```

Comment:
Have you checked if the tests will run correctly and also the local cucumber HTML reports will work if RUN_IN_CI=true?

Reply:
Yes anand. PFA the Screenshot with RUN_IN_CI=true
<img width="1512" alt="image" src="https://github.com/znsio/teswiz/assets/64599820/820487ad-b186-4832-b082-66b58c9c797c">